shuup_tests/core/test_admin_menu.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# This file is part of Shuup.
+#
+# Copyright (c) 2012-2019, Shoop Commerce Ltd. All rights reserved.
+#
+# This source code is licensed under the OSL-3.0 license found in the
+# LICENSE file in the root directory of this source tree.
+from __future__ import unicode_literals
+
+from shuup.admin.menu import MAIN_MENU
+from shuup.testing.admin_main_menu_updater import SubscriptionMainMenuUpdater
+
+
+def test_menu_updater():
+    updater = SubscriptionMainMenuUpdater(MAIN_MENU)
```

Comment:
Use the "overrride_provides" instead. First test the "get menu" without this "menu updater" and then with and make sure those updates actually happened.

Reply:
fixed